model/patch/patch.go

```diff
@@ -639,13 +656,7 @@ func (p *Patch) GetRequester() string {
 }
 
 func (p *Patch) CanEnqueueToCommitQueue() bool {
-	for _, modulePatch := range p.Patches {
-		if !modulePatch.IsMbox {
-			return false
-		}
-	}
-
-	return true
+	return p.GitInfo != nil
```

Comment:
Should this also check that username / email are populated? Otherwise I'm not sure another function is necessary

Reply:
done